internal/filesystem.go

```diff
@@ -343,6 +346,7 @@ func CliDimensions() ([]string, error) {
 func IsServiceActive(service string) bool {
 	out, err := exec.Command(SystemctlExec, "is-active", service).Output()
 	if err != nil {
+		log.Println(WarningPrefix, "service:", service, " is-active check result:", err)
```

Comment:
same here?

Reply:
To be removed. See last comment to @piotrjurkiewicz feedback.